src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/TrigFunctions.txt

```diff
@@ -0,0 +1,351 @@
+﻿>> Sin(0)
+0
+
+>> Sin(Pi())
+0
+
+>> Sin(Pi()/2)
+1
+
+>> Sin(-Pi()/2)
+-1
+
+>> Sin(3*Pi()/2)
+-1
+
+>> Sin(2*Pi())
+-0
+
+>> Sin(Pi()/4)
+0.707106781
+
+>> Sin(-Pi()/4)
+-0.707106781
+
+>> Sin(Blank())
+0
+
+>> Sin(Table({a:0},{a:Pi()/6},{a:Pi()/2}))
+#skip
+// Tabular version not implemented yet: expected: Table({a:0},{a:0.5},{a:1})
+
+>> Sin(Table({a:0},{a:1/0},{a:Pi()/2}))
+#skip
```

Comment:
Can you move tabular to a separate file?
(easier to exclude for other backends that don't support tabular)

Reply:
Done